sites/default/statement.inc.php

```diff
@@ -102,278 +463,280 @@
 //  [Your billing dept phone]
 
 function create_statement($stmt) {
```

Comment:
I added the $GLOBALS['billing_phone_number'] since the default is to pull the phone number from the practice data and my billing people are not local - they are an outside group with their own phone number.  BTW they are really good using openEMR/Trizetto.  Their big complaint was the look and feel of the statement, which they send out directly.  Theoretically, the address where to send the payments could also be different...

The changes above are to get the return address lined up for a standard windowed envelope -the addresses and payment information positions are swapped in the bottom third of the statement - and to have a balance due field in this "tear-off" portion.
 
 I may need to tweak the lines a little but that is the thinking.